ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -369,6 +374,52 @@ public sealed class ArchiWebHandler : IDisposable {
 			return result;
 		}
 
+		[PublicAPI]
+		public async Task<int?> GetPointsBalance() {
+			(bool success, string? accesstoken) = await CachedAccessToken.GetValue().ConfigureAwait(false);
+
+			if (!success || string.IsNullOrEmpty(accesstoken)) {
+				return null;
+			}
+
+			KeyValue? response = null;
+
+			for (byte i = 0; (i < WebBrowser.MaxTries) && (response == null); i++) {
+				using WebAPI.AsyncInterface loyaltyRewardsService = Bot.SteamConfiguration.GetAsyncWebAPIInterface(LoyaltyRewardsService);
+
+				loyaltyRewardsService.Timeout = WebBrowser.Timeout;
+
+				try {
+					response = await WebLimitRequest(
+						WebAPI.DefaultBaseAddress,
+
+						// ReSharper disable once AccessToDisposedClosure
+						async () => await loyaltyRewardsService.CallAsync(
+							HttpMethod.Get, "GetSummary", args: new Dictionary<string, object>(2, StringComparer.Ordinal) {
+								{ "access_token", accesstoken! },
+								{ "steamid", Bot.SteamID.ToString(CultureInfo.InvariantCulture) }
+							}
+						).ConfigureAwait(false)
+					).ConfigureAwait(false);
+				} catch (TaskCanceledException e) {
+					Bot.ArchiLogger.LogGenericDebuggingException(e);
+				} catch (Exception e) {
+					Bot.ArchiLogger.LogGenericWarningException(e);
+				}
+			}
+
+			if (response == null) {
+				Bot.ArchiLogger.LogGenericWarning(string.Format(CultureInfo.CurrentCulture, Strings.ErrorRequestFailedTooManyTimes, WebBrowser.MaxTries));
+				return null;
+			}
+
+			if (string.IsNullOrEmpty(response["summary"]["points"].Value)) {
```

Comment:
This is wrong check, you should compare it to `KeyValue.Invalid` or something like that in order to check if this exists, not the value, I should have similar checks in other ASF places.